src/DecileStatistics.java

```diff
@@ -0,0 +1,41 @@
+import java.util.*;
+
+public class DecileStatistics {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
```

Comment:
main method is too long extract some code to methods .

Reply:
Done.